src/async_hta_service.hpp

```diff
@@ -137,6 +143,17 @@ class AsyncHtaService
             }
         }
 
+        try
+        {
+            auto logging = config.at("logging");
+            logging_.log_nan_values = logging.at("nan_values").get<bool>();
+            logging_.log_non_monotonic_values = logging.at("non_monotonic_values").get<bool>();
+        }
+        catch (std::exception& e)
+        {
+            Log::info() << "Couldn't parse logging section of the config: " << e.what();
+        }
```

Comment:
This should be backward compatible w.r.t. missing `logging` section and missing keys within it.